docs/agent-k8s-install-helm.md

```diff
@@ -0,0 +1,117 @@
+# Install Using Helm
+
+Use the Helm package manager to install the Smart Agent to Kubernetes
+environments.
+
+If you don't have Helm installed already, use kubectl
+to install the Smart Agent. To learn more, see
+[Install the Smart Agent using kubectl](agent-k8s-install-kubectl.md).
+
+## Prerequisites
+
+* Linux kernel version 2.6.32 or higher
+* cap_dac_read_search and cap_sys_ptrace capabilities
+* Terminal or a similar command-line interface application
+* Helm:
+
+  - The installation chart for the Smart Agent is compatible with Helm version 2 or Helm version 3.
+  - To learn more, see the [Helm site](https://helm.sh/).
+
+* Tiller installed on all of your Kubernetes hosts. Because of
+  the way that the installation chart works, you need Tiller even if you're using
```

Comment:
Says who? Tiller is gone in Helm 3 and I can confirm is totally unnecessary if using Helm 3+.

Reply:
Docs currently say that Tiller is required. See https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.kubernetes.html.

I can remove it if you say so.